test/built-ins/Temporal/PlainTime/prototype/add/precision-exact-mathematical-values-2.js

```diff
@@ -0,0 +1,23 @@
+// Copyright (C) 2022 André Bargull. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal.plaintime.prototype.add
+description: >
+  Duration components are precise mathematical integers.
+includes: [temporalHelpers.js]
+features: [Temporal]
+---*/
+
+let duration = Temporal.Duration.from({
+  hours: Number.MAX_VALUE,
+  minutes: Number.MAX_VALUE,
+});
+
+let time = new Temporal.PlainTime(0, 0, 0, 0, 0, 0);
+
+let result = time.add(duration);
+
+TemporalHelpers.assertPlainTime(result, 10, 8, 0, 0, 0, 0);
+
+assert.sameValue("10:08:00", result.toString());
```

Comment:
Actually I've removed this line, because it was a leftover from the original test and the assertion is superseded by `TemporalHelpers.assertPlainTime`.